docs/src/explanations/properties.md

```diff
@@ -143,6 +143,7 @@ class IntegerArithmeticExample extends RawModule {
 The following table lists the possible arithmetic operators that are supported
 on integral `Property` typed values.
 
-| Operation | Description                                                |
-| --------- | -----------                                                |
-| `+`       | Perform addition as defined by FIRRTL spec section 25.1.1. |
+| Operation | Description                                                      |
+| --------- | -----------                                                      |
+| `+`       | Perform addition as defined by FIRRTL spec section 25.1.1.       |
```

Comment:
nit: it seems likely that these section numbers would move around; are we planning to keep updating this? Maybe we can refer to the section by name instead which seems likely to be more stable?

Reply:
hmm yeah, they probably will drift. i think referencing by name sounds good.